doc/Tex/RAM_SCB_intro.tex

```diff
@@ -1,56 +1,17 @@
 
 \section{About This Manual}
-Users who want to install and begin quickly should start at Chapter 
-\ref{chp:quick}, which quickly outlines the path from installation to 
-simulation with little detail.  The installation process is discussed fully 
-in Chapter \ref{chp:install}.  Instructions on performing simulations, 
-as well as several example simulations, are given in Chapter \ref{chp:run}. 
- An outline of using this code in the Space Weather Modeling Framework is 
-found in Chapter \ref{chp:swmf}. Useful scripts included in the distribution
-are described, in brief, in Chapter \ref{chp:scripts}.
-Finally, a complete list of all param file commands is found in Chapter \ref{chp:param}.
+Users who want to install and begin quickly should start at Chapter \ref{chp:quick}, which quickly outlines the path from installation to simulation with little detail.  The installation process is discussed fully in Chapter \ref{chp:install}.  Instructions on performing simulations, as well as several example simulations, are given in Chapter \ref{chp:run}. An outline of using this code in the Space Weather Modeling Framework is found in Chapter \ref{chp:swmf}. Useful scripts included in the distribution are described, in brief, in Chapter \ref{chp:scripts}. Finally, a complete list of all param file commands is found in Chapter \ref{chp:param}.
 
 \section{TERMS OF USE \& DISTRIBUTION POLICY}
 
-Use of the RAM-SCB software implies agreement with the terms herein.
-RAM-SCB is open source software that has been developed at Los Alamos National Laboratory (LANL). The code is based on physics and
-numerical methods detailed in the following publications and references therein:
+Use of the RAM-SCB software implies agreement with the terms herein. RAM-SCB is open source software that has been developed at Los Alamos National Laboratory (LANL). The code is based on physics and numerical methods detailed in the following publications and references therein:
 
-1. Jordanova, V. K. et al. (2006), Kinetic simulations of ring current 
-evolution during the Geospace Environment Modeling challenge events, 
-J. Geophys, Res., 111, A11S10, doi:10.1029/2006JA011644. \\
+1. Jordanova, V. K. et al. (2006), Kinetic simulations of ring current evolution during the Geospace Environment Modeling challenge events, J. Geophys, Res., 111, A11S10, doi:10.1029/2006JA011644. \\

```

Comment:
These need updating. Recent Welling publication, plus the Engel paper. As you know, a lot has changed and those papers aren't really up-to-date descriptions any more.